spec/spec_helper.cr

```diff
@@ -1,3 +1,7 @@
+Granite.settings.database_url = ENV["MYSQL_DATABASE_URL"]
```

Comment:
How does this work without overwriting the previous connection? :confused:

Reply:
_magic_

https://github.com/Blacksmoke16/granite-orm/blob/database_url-deprecations/src/granite/settings.cr#L12-L23

the database_url= method is just a wrapper that in the backgrounds sets the given URL as the value with the adapter being the key.  Since each of those uses a different adapter, it saves all three and doesn't override anything.

Reply:
Too much magic for me :( I'm boring :sweat_smile: